plugins/ptp_operator/metrics/ptp4lParse.go

```diff
@@ -174,3 +179,30 @@ func handleHoldOverState(ptpManager *PTPEventManager,
 		}
 	}
 }
+
+func isFollowerOnly(ptp4lCfg *ptp4lconf.PTP4lConfig) bool {
+	if section, ok1 := ptp4lCfg.Sections["global"]; ok1 {
+		if value, ok2 := section["slaveOnly"]; ok2 {
+			log.Errorf("FollowerOnly scenario detected")
```

Comment:
this shouldn't be an error. I guess it is temporary?

Reply:
Fixed